src/components/shared/btn-reset.tsx

```
@@ -0,0 +1,27 @@
+  onClick: () => void;
+  label?: string;
+  className?: string;
+}
+
+export default function BtnReset({ onClick, label = "Reset", className }: ResetButtonProps) {
+    const t = useTranslations("reset");
+  
+  return (
+    <Button
+      onClick={onClick}
+      className={cn("flex items-center gap-1 text-red-600 hover:text-red-700 bg-transparent hover:bg-transparent transition text-sm font-normal",className)}
```

ESLint

[error] A space is required after ','.

(comma-spacing)